ui/app/components/Breadcumb.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import type { RouteLocationNormalizedLoaded } from "vue-router";
+
+const route: RouteLocationNormalizedLoaded = useRoute();
```

Comment:
This is ok, but this particular type is auto inferred. this is true for all macros and utilities, we don't need to explicitly assign types.